packages/browsers/README.md

```diff
@@ -40,3 +40,13 @@ Known limitations:
 ## API
 
 The programmatic API allows installing and launching browsers from your code. See the `test` folder for examples on how to use the `install`, `canInstall`, `launch`, `computeExecutablePath`, `computeSystemExecutablePath` and other methods.
+
+## Testing instructions
+
+To test the package download the browser binaries to the cache folder first:
```

Comment:
Can we make this part of `test` command? We already skip the downloading if the file path exits.

Note: Testing should have a parent header with name of `Contributing`